coderd/database/dbauthz/dbauthz.go

```diff
@@ -2240,6 +2330,13 @@ func (q *querier) GetReplicasUpdatedAfter(ctx context.Context, updatedAt time.Ti
 	return q.db.GetReplicasUpdatedAfter(ctx, updatedAt)
 }
 
+func (q *querier) GetRunningPrebuiltWorkspaces(ctx context.Context) ([]database.GetRunningPrebuiltWorkspacesRow, error) {
+	if err := q.authorizeContext(ctx, policy.ActionRead, rbac.ResourceSystem); err != nil {
```

Comment:
As mentioned above, `ResourceSystem` should be avoided.

Is there some reason we can't / shouldn't just use the normal rules for reading Workspaces as the access control for this query?